openage/convert/main.py

```diff
@@ -159,25 +171,45 @@ def acquire_conversion_source_dir():
 
     if 'AGE2DIR' in os.environ:
         sourcedir = os.environ['AGE2DIR']
-        print("using AGE2DIR: " + sourcedir)
+        print("found environment variable AGE2DIR")
     else:
-        # TODO use some sort of GTK GUI for this.
-        print("please provide your AoE II installation dir:")
+        # TODO: use some sort of GUI for this (GTK, QtQuick, zenity?)
+        proposals = [proposal for proposal in _get_source_dir_proposals()
+                     if os.path.exists(_expand_relative_path(proposal))]
+        print("Select an Age of Kings installation directory. "
+              "Insert the index of one of the proposals, or any path:")
+        for index, proposal in enumerate(proposals):
+            print("({}) {}".format(index, proposal))
 
         try:
-            sourcedir = input("> ")
+            user_selection = input("> ")
+            if user_selection.isdecimal():
+                sourcedir = proposals[int(user_selection)]
+            else:
+                sourcedir = user_selection
         except KeyboardInterrupt:
-            print("")
+            print("Interrupted, aborting")
             exit(0)
         except EOFError:
-            print("")
+            print("EOF, aborting")
```

Comment:
It seemed useful to me when trying it out. And it tells more than an empty print. If you don't like the message, I suggest deleting the print altogether